WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -1304,17 +1334,41 @@ public void run() {
     @SuppressWarnings("unused")
     @Subscribe(threadMode = ThreadMode.MAIN)
     public void onSiteChanged(OnSiteChanged event) {
-        // We should be safe to ignore the errors
-        if (isFinishing() && event.isError()) {
+        if (isFinishing()) {
             return;
         }
-        mSite = mSiteStore.getSiteBySiteId(mSite.getSiteId());
 
-        // We try to fetch the site after Automated Transfer is completed so that we can fetch its plugins. If
-        // we are still showing the AT progress and the site is AT site, we can continue with plugins fetch
-        if (mIsShowingAutomatedTransferProgress && mSite.isAutomatedTransfer()) {
-            mDispatcher.dispatch(PluginActionBuilder.newFetchPluginDirectoryAction(new PluginStore
-                    .FetchPluginDirectoryPayload(PluginDirectoryType.SITE, mSite, false)));
+        if (!event.isError()) {
+            mSite = mSiteStore.getSiteBySiteId(mSite.getSiteId());
+        } else if (mIsShowingAutomatedTransferProgress) {
+            AppLog.e(T.PLUGINS, "Fetching the site after Automated Transfer has failed with error type "
```

Comment:
Do we need to do anything more here to handle the error?

Reply:
Nope, we only trigger a site fetch action for Automated Transfer and we shouldn't even receive the `onSiteChanged` event otherwise. Any other error should be handled in a more centralized place, which I am not sure if we are doing or not.